guestbook/discovery/model.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: gbapp
+  annotations:
+    core.hybridapp.io/hybrid-discovery: "enabled"
+    tools.hybridapp.io/hybrid-discover-clusterscoped: "true"
```

Comment:
for consistency, let's use `hybrid-discovery-clusterscoped` instead of `hybrid-discover-clusterscoped` here. I'll make the changes in the projects in a separate PR